lib/actions/preload.coffee

```diff
@@ -163,8 +169,7 @@ module.exports =
 		.then (docker) ->
 
 			# Build the preloader image
-			buildOutputStream = preload.build(docker)
-			buildOutputStream.pipe(process.stdout)
```

Comment:
Does this output ever contain anything useful? Might be worth keeping this pipe, but only in debug mode (i.e. `if process.env.DEBUG`)